experiments/simple_example_mnist_loss/.gitignore

```diff
@@ -0,0 +1 @@
+!plot_loss.png
```

Comment:
do we need this or can we rely on the global gitignore?

Reply:
I put the plot_loss.png in my own public [gist](https://gist.github.com/danielsnider/73519752010e430af55380f9bf2ee653) and linked it in the README.md. If that is okay, it leaves us with two fewer files in the repo the image and this .gitignore file👍.